### ISSUE TITLE
how to emit a socket event after updating a document?

### ISSUE THREAD

**Issue by jitesh121** [angular-fullstack/generator-angular-fullstack#2784]
- [ ] I understand that GitHub issues are not for tech support, but for questions specific to this generator, bug reports, and feature requests.

Item | Version
----- | -----
generator-angular-fullstack | x.x.x
Node | x.x.x
npm | x.x.x
Operating System | OS X 10 / Windows 10 / Ubuntu 15.10 / etc
etc | etc

Item | Answer
----- | -----
Transpiler | Babel / TypeScript
Markup | HTML / Pug
CSS | CSS / LESS / SCSS / Stylus
Router | ngRoute / ui-router
Client Tests | Jasmine / Mocha
DB | MongoDB / SQL
Auth | Y / N
etc | etc

**Comment by lilian131** [comment]
hi @jitesh121 did you figure out ?

**Comment by jitesh121** [comment]
yes , i figured it out @lilian131

**Comment by lilian131** [comment]
Yes me too it does'nt send the event if we use findoneandupdate or updateone or ....
it works if after findoneandupdate i use item.save again
